src/components/tutorial-meta/index.tsx

```diff
@@ -56,15 +56,23 @@ export default function TutorialMeta({
 						isInteractive,
 					}}
 				/>
-				<span className={s.buttonGroup}>
-					<InteractiveLabButton />
-					{/** // NOTE! - hiding this component from prod until auth is enabled  */}
-					{isAuthEnabled ? (
-						<TutorialMetaBookmarkButton
-							tutorial={{ id: tutorialId, name: heading.text }}
-						/>
-					) : null}
-				</span>
+				{/**
+				 * TODO: remove this conditional once auth is released to prod,
+				 * since the bookmark button will always render. If we don't
+				 * conditionally render this and there is no interactive lab button,
+				 * a margin stacking occurs that disrupts the layout
+				 */}
+				{isInteractive || isAuthEnabled ? (
+					<span className={s.buttonGroup}>
+						<InteractiveLabButton />
```

Comment:
I think we need to wrap this in `isInteractive` now since this block can be triggered if `isInteractive` is false and `isAuthEnabled` is true? 🤔

Reply:
But if `isAuthEnabled` is true then the bookmark button will render which is okay!

Reply:
Basically we don't want this wrapper div to render if there isn't a child button

Reply:
The interactive lab button won't render if there isn't an active lab in the context. So that is handled elsewhere

Reply:
Yes, true! But so will the `InteractiveLabButton` even if `isInteractive` is false.

Here's what I'm meaning to suggest:

```typescript
{isInteractive || isAuthEnabled ? (
	<span className={s.buttonGroup}>
		{isInteractive ? <InteractiveLabButton /> : null}
		{/*// NOTE! - hiding this component from prod until auth is enabled */}
		{isAuthEnabled ? (
			<TutorialMetaBookmarkButton
				tutorial={{ id: tutorialId, name: heading.text }}
			/>
		) : null}
	</span>
) : null}
```

Reply:
Oops! Didn't refresh before sending that. I see what you're saying! `InteractiveLabButton` has this:

```typescript
if (!ctx.labId) {
	return null
}
```